From 00a1f5c96c2a763ded481e2f387d4f1624c91249 Mon Sep 17 00:00:00 2001
From: Tianshu Qiu <tian.shu.qiu@intel.com>
Date: Wed, 30 May 2018 11:10:47 +0800
Subject: [PATCH 1489/1569] media: intel-ipu4p: imx355: Correct link frequency

Message for Open Source:
The MIPI link frequency for all supported modes
are 360MHz.

Message for Internal:
[Issue/Feature] Refer to open source message.
[Root Cause/Changes] Correct link frequency.

Change-Id: I7e33d4a9cc3c6f214384246b3f2a6fcef2808626
Signed-off-by: Tianshu Qiu <tian.shu.qiu@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/i2c/imx355.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/i2c/imx355.c b/drivers/media/i2c/imx355.c
index b0b5c82..9bdd65a 100644
--- a/drivers/media/i2c/imx355.c
+++ b/drivers/media/i2c/imx355.c
@@ -875,7 +875,7 @@ static const int imx355_test_pattern_val[] = {
 /* Configurations for supported link frequencies */
 /* Menu items for LINK_FREQ V4L2 control */
 static const s64 link_freq_menu_items[] = {
-	300800000,
+	360000000,
 };
 
 /* Mode configs */
-- 
2.7.4

